docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -373,27 +355,48 @@ The information is interesting for various purposes, including running Otterize
 Client intents are the cornerstone of [intent-based access control](https://otterize.com/ibac).
 :::
 
-2. Keep an eye on the logs being tailed in the **[client-other]** while you apply this `intents.yaml` file in your **main terminal window** using:
+We can apply intents for the `client` by applying the `client-intents.yaml` file:
+
 ```bash
 kubectl apply -f https://docs.otterize.com/code-examples/kafka-mtls/client-intents.yaml
 ```
 
-You should quickly see in the **[client-other]** that the **other client cannot** access the topic:
-```
-Sent message - Message 12 [sent by client-other]            # <- before applying the intents file
-Sent message - Message 13 [sent by client-other]            # <- before applying the intents file
-time="2022-10-06T09:44:53Z" level=info error="kafka server: # <- after applying the intents file
- The client is not authorized to access this topic"
-Loop exited
-```
-Meanwhile, in the **[client]** terminal you can see that the **client can** access the topic:
+If you go back to your access graph, you'll now see that the `client` has a solid green arrow connecting it to the Kafka broker. This is because the `client` has both declared its intent to access the Kafka broker, and it has been authenticated using mTLS.
+
+If you click on that solid line, you will see that the declared intents match the discovered intents, so access is assured.
+
+![client intents applied](../../static/img/quick-tutorials/k8s-kafka-mtls/client-intents.png)
+
+2. At this point, since the Kafka server is not actually protected, the `client-other` can still access the topics. The line is dashed, not solid, indicating that it has no declared intents.
+
+![Declared Intent](../../static/img/quick-tutorials/k8s-kafka-mtls/declared-intent.png)
+
+3. Now let's make a change to the `kafkaserverconfig` to require intents for the 2 topics.
+
+```bash
+kubectl apply -f https://docs.otterize.com/code-examples/kafka-mtls/kafkaserverconfig-protect.yaml
 ```
-Sent message - Message 24 [sent by client]
-Read message - Message 24 [sent by client]
-Sent message - Message 25 [sent by client]
-Read message - Message 25 [sent by client]
+
+<details>
+<summary>Expand to see kafkaserverconfig-protect</summary>
```

Comment:
```suggestion
<summary>See changes needed to enable intentsRequired</summary>
```
And once more I would do it expanded - with no collapsible, and highlight the changed lines.

Reply:
This whole section is no longer required, since we started off with `true` `true` so I'm deleteing it all.